cmd/habitatctl/commands/start.go

```diff
@@ -24,20 +24,26 @@ func parseFlags(args []string) ([]string, []string) {
 
 // startCmd represents the start command
 var startCmd = &cobra.Command{
-	Use:                "start [process] [env vars as VAR=value] [flags",
-	Short:              "Starts a habitat process",
-	Long:               `TODO create long description`,
-	FParseErrWhitelist: cobra.FParseErrWhitelist{UnknownFlags: true},
-	DisableFlagParsing: true,
-	Args:               cobra.MinimumNArgs(1),
+	Use:   "start [process] -- [env vars] [flags]",
+	Short: "Starts a habitat process",
+	Long:  `TODO create long description`,
+	Args:  cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
+		process := args[0]
+		fmt.Println("start process ", process)
```

Comment:
I like that we're using -- to separate args to the app, it lets us have flags on habitatctl start that don't get passed through which is good in case we ever need it. Could we squash this commit into the previous one, since it undoes some of the changes the last one made?